scripts/variantstore/wdl/GvsExtractCallset.wdl

```diff
@@ -16,10 +15,11 @@ workflow GvsExtractCallset {
         String fq_cohort_extract_table_prefix
         String query_project = data_project
 
-        String fq_filter_set_info_table = "~{data_project}.~{default_dataset}.filter_set_info"
-        String fq_filter_set_site_table = "~{data_project}.~{default_dataset}.filter_set_sites"
-        String fq_filter_set_tranches_table = "~{data_project}.~{default_dataset}.filter_set_tranches"
         Boolean do_not_filter_override = false
+        String? filter_set_name
```

Comment:
Filter set name should not be optional. This will be named something that indicates which release of the filtering model the extract should use.

Reply:
should it be set even if filtering is not going to be used? for example, if do_not_filter_override is true, the argument will never be used

Reply:
oh, i didn't catch that. It's fine to be optional.